src/qualimap/qualimap_rnaseq/script.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set -eo pipefail
+
+tmp_dir=$(mktemp -d -p "$meta_temp_dir" qualimap_XXXXXXXXX)
+
+# Handle output parameters
+if [ -n "$par_output_report" ]; then
+    outfile=$(basename "$par_output_report")
+    report_extension="${outfile##*.}"
+fi
+
+if [ -n "$par_output_counts" ]; then
+    counts=$(basename "$par_output_counts")
+fi
+
+# disable flags
+[[ "$par_paired" == "false" ]] && unset par_paired
+[[ "$par_sorted" == "false" ]] && unset par_sorted
+
+# Run qualimap
+qualimap rnaseq \
+    --java-mem-size=$par_java_memory_size \
```

Comment:
If you set this to the following, does this work?

```suggestion
    ${meta_memory_mb:+--java-mem-size=${meta_memory_mb}M} \